eth_keys/backends/native/ecdsa.py

```diff
@@ -60,6 +60,35 @@ def private_key_to_public_key(private_key_bytes: bytes) -> bytes:
     return public_key_bytes
 
 
+def compress_public_key(uncompressed_public_key_bytes: bytes) -> bytes:
+    x, y = decode_public_key(uncompressed_public_key_bytes)
+    if y % 2 == 0:
+        prefix = b"\x02"
+    else:
+        prefix = b"\x03"
+    return prefix + pad32(int_to_big_endian(x))
```

Comment:
lol, padding the compressed key is a strange spec. I was wondering why the compressed key is always the same length...